internal/pkg/cli/svc_package.go

```diff
@@ -117,6 +120,22 @@ func newPackageSvcOpts(vars packageSvcVars) (*packageSvcOpts, error) {
 	return opts, nil
 }
 
+func minifyRawManifest(ws manifestReader, workloadName string) ([]byte, error) {
+	raw, err := ws.ReadWorkloadManifest(workloadName)
+	if err != nil {
+		return nil, fmt.Errorf("read manifest file for %s: %w", workloadName, err)
+	}
+	var cleaned any
```

Comment:
I wonder if we should keep those comments because they won't add too much to the manifest size and when users do `svc show --manifest` we could make sure they have the same manifest experience (proper comments for explanation).

Reply:
Done!